Remotion/Development/Web/ResourceHosting/ResourceVirtualPathProvider.cs

```diff
@@ -59,7 +56,7 @@ public ResourceVirtualPathProvider (ResourcePathMapping[] mappings, FileExtensio
     {
       ArgumentUtility.CheckNotNull("mappings", mappings);
 
-      _resourceRoot = VirtualPathUtility.AppendTrailingSlash(CombineVirtualPath("~/", WebConfiguration.Current.Resources.Root));
+      _resourceRoot = VirtualPathUtility.AppendTrailingSlash(CombineVirtualPath("~/", SafeServiceLocator.Current.GetInstance<ResourceRoot>().Value));
```

Comment:
It feels like we should put the ResourceRoot into a local variable, i.e. separate the statements to improve readability.